### DISCUSSION TITLE
"No permission" - Page for Group "Guest"

### DISCUSSION THREAD

**Discussion by h3adbang3r** [TandoorRecipes/recipes#2871] [Q&A]
Hello,

I am using TandoorRecipes in a Docker container on an Unraid system. I have a tablet in my kitchen and want to prevent changes. Therefore, I created a user account and placed it in the "Guest" group. I have also added this account to all cookbooks.

Now, the issue:
When I click on books in the bottom menu bar, I receive a message:

"No permission

You do not have the necessary permissions to view this page or perform this action. Please contact an administrator."

Even manually assigning viewer rights for cookbooks, etc., in Django does not improve it. It seems that the permissions for the links "Books," "Meal Plan," and "Shopping List" are not set.

Thank you for your assistance, and greetings from Berlin.

**Comment by vabene1111** [comment]
Hi, guests are not supposed to use those functions. The permission system is very rudimentary but that is the way it is for now.

This needs to be improved in the future.

Greetings from Berlin as well :)
